specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/examples/Attestations_CreateSubscriptionScope_AllProperties.json

```diff
@@ -0,0 +1,89 @@
+{
+  "parameters": {
+    "subscriptionId": "35ee058e-5fa0-414c-8145-3ebb8d09b6e2",
+    "api-version": "2019-10-01",
+    "attestationName": "790996e6-9871-4b1f-9cd9-ec42cd6ced1e",
+    "parameters": {
+      "properties": {
+        "policyAssignmentId": "/subscriptions/35ee058e-5fa0-414c-8145-3ebb8d09b6e2/providers/microsoft.authorization/policyassignments/b101830944f246d8a14088c5",
+        "policyDefinitionReferenceId": "0b158b46-ff42-4799-8e39-08a5c23b4551",
+        "complianceState": "Compliant",
+        "expiresOn": "2021-06-15T00:00:00Z",
+        "owner": "55a32e28-3aa5-4eea-9b5a-4cd85153b966",
```

Comment:
What does owner signify? Will this map to createdBy?

Reply:
Owner is a designated person that someone is saying "owns" setting the state for the resource. It is a free-form field that can be any reasonable length string and is not necessarily the same person that created/lastmodified the attestation.

Reply:
I think the description for owner is OK.  If you think it needs work let me know.

Reply:
I think we can improve the description, is this UserID, username, may be an example would help.